modules/Makefile

```diff
@@ -144,7 +143,8 @@ PACKAGES_TO_DOCUMENT = \
 	packages/UnitTest.chpl \
 	packages/UnorderedAtomics.chpl \
 	packages/UnorderedCopy.chpl \
-	packages/URL.chpl
+	packages/URL.chpl \
```

Comment:
Adding the backslash on the last line is a little off-putting to me, but I also see the advantage of consistency.

Reply:
Yeah. I did this to make it easier to sort these in the editor. (I think it's similar to the ability to write a trailing comma on an enum, which it seems not everybody likes).